Makefile

```diff
@@ -0,0 +1,610 @@
+#
+# name_of_the_rule : | input files where only existence matters
+#     one commands
+#     more commands
+#
+# Running Makefiles
+# ------------------------------------------------------------------------------
+#
+# To run a named rule, you would then do:
+#
+# $ make name_of_the_rule
+#
+# For normal rules, you would hit:
+#
+# $ make <outputfile>
+#
+# Tip: Type "make" and hit tab twice in the shell, to show available targets to
+# run in the current makefile.
+#
+# Special variables and more
+# ------------------------------------------------------------------------------
+#
+# Inside the commands of the rules, one can write pretty much bash, especially
+# if setting `SHELL := /bin/bash` in the beginning of the Makefile which we have
+# done below.
+#
+# There are a some differences though:
+#
+# 1. Variable syntax using only a single $-character refer to MAKE-variables.
+#    Thus, to use variables set in bash, you have to use double-$.
+#    So:
+#    echo $(this-is-a-make-variable) and $$(this-is-a-bash-variable)
+#
+#    This also goes for command substitution, so rather than:
+#
+#    	echo "Lines in file:" $(wc -l somefile.txt)
+#
+#    ... you would write:
+#
+#    	echo "Lines in file:" $$(wc -l somefile.txt)
+#
+# 2. Makefiles also use some special variables with special meaning, to access
+#    things like the output and input files:
+#
+#    $@   The output target (In case of multiple targets, the same command will
+#         be run once for every output file, feeding this variable with only a
+#         single one at a time)
+#
+#    $<   The first input file or dependency
+#
+#    $^   ALL input files / dependencies.
+#
+# 3. Another speciality is that adding a '@' character before any command, will
+#    stop this command from being printed out when it is executed (only its output
+#    will be printed).
+#
+# That's all for now. For the official docs, see:
+# https://www.gnu.org/software/make/manual/make.html
+#
+# Some more caveats
+# ------------------------------------------------------------------------------
+#  Here are some things that might not be obvious at first:
+#
+#  To create a rule with a single output but many dependencies, you can add
+#  these dependencies on multiple lines by using the continuation character \
+#  like so:
+#
+#  name_of_rule: dependency1 \
+#  		dependency2 \
+#  		dependency3 \
+#  		dependency4 \
+
+# ==============================================================================
+# Various definitions
+# ==============================================================================
+# Make sure bash is used as shell, for consistency and to make some more
+# advanced scripting possible than with /bin/sh
+SHELL := /bin/bash
+
+# Define path variables
+SCRIPT_DIR := $(shell pwd)
+ASSETS_DIR := $(shell realpath $(SCRIPT_DIR)/assets/)
+CONTAINER_DIR := $(realpath $(SCRIPT_DIR)/container/)
+PRODIGAL_TRAINING_DIR := $(ASSETS_DIR)/prodigal_training_files
```

Comment:
With the update of #192 this will change to:
```
PRODIGAL_TRAINING_DIR := $(ASSETS_DIR)/chewBBACA/CHEWBBACA/prodigal_training_files
```

Reply:
Nevermind. I decided to just fetch the `.ptf` files and put them in the `prodigal_training_files` dir